portfolio/src/main/webapp/index.html

```diff
@@ -143,24 +141,35 @@ <h2 id = "gallery" > Gallery </h2>
                 <img class = "slide" src="/images/group3.jpg">
             </div>
 
+            <div class = "mySlides">
+                <img class = "slide" src="/images/canada.jpg">
+            </div>
+
+            <div class = "mySlides">
+                <img class = "slide" src="/images/littlepecks2.jpg">
+            </div>
+
             <a class = "prev" onclick="plusSlides(-1)">❮</a>
             <a class = "next" onclick="plusSlides(1)">❯</a>
 
         </div>
        
       <h2 id = "contact" > Contact </h2>
+        <div class = "clearfix">
+        <a class="fa fa-google"></a>
```

Comment:
There doesn't seem to be an `fa` class anymore, so it should be removed here and for linkedin.

Reply:
clearfix is a term i got from w3schools I use it whenever I want to align my image and text side by side. I tried removing fa but the fa class is used by the icon stylesheet that I added at the top so if I remove it it alters the icon appearance

Reply:
Ah got it now, sorry for my confusion.